src/main/java/com/google/maps/model/LatLng.java

```diff
@@ -50,6 +50,6 @@ public String toString() {
   @Override
   public String toUrlValue() {
     // Enforce Locale to English for double to string conversion
-    return String.format(Locale.ENGLISH, "%f,%f", lat, lng);
+    return String.format(Locale.ENGLISH, "%s,%s", lat, lng);
```

Comment:
I believe this will fail in certain European countries, where the decimal point is replaced with a comma in locale rendered numbers. We need the number to be in english format for the API server to grok the number. 

Can you raise an issue that demonstrates the precision issue you are having instead of just supplying a fix?